src/app/shared/model/audio-version.model.ts

```diff
@@ -44,13 +44,13 @@ export class AudioVersionModel extends BaseModel implements HasUpload {
     this.setLabel();
   }
 
-  setLabel() {
+  private setLabel() {
     if (this.doc && this.doc['label']) {
-      this.set('label', this.doc['label']); // probably already set to this
+      this.set('label', this.doc['label'], true); // probably already set to this
```

Comment:
This all feels right to me.  New AudioVersions should look like they haven't `changed()`, so we should force-set the labels on them.